### DISCUSSION TITLE
Qodana report is empty on large Kotlin project

### DISCUSSION THREAD

**Discussion by sschuberth** [JetBrains/Qodana#125] [Q&A] [answered]
I started to evaluate Qodana on the code base of the [OSS Review Toolkit](https://github.com/oss-review-toolkit/ort) and created a [draft PR](https://github.com/oss-review-toolkit/ort/pull/5722) that leverages the GitHub Action. Surprisingly, the [summary shows no issues](https://github.com/oss-review-toolkit/ort/actions/runs/2966144159#summary-8120318910) at all. While that's basically nice (😁) it's unfortunately unrealistic, as we e.g. have several open inspections hints in IntelliJ. Any idea what's going on here?

**Comment by brichbash** [comment, marked as answer]
Hello,
By default, the action analyzes only changes in PRs, so that's the reason Qodana didn't report anything.
If you want Qodana to analyze all files in repo, please set `pr-mode` to `false` in the action configuration. [Here](https://github.com/JetBrains/qodana-action#configuration) you can find other available configuration parameters.
